tools/distrib/python/submit.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import shutil
+import subprocess
+import tempfile
+
+parser = argparse.ArgumentParser(
+    description='Submit the package to a PyPI repository.')
+parser.add_argument(
+    '--repository', metavar='r', type=str, default='pypi',
+    help='The repository to push the package to. '
+         'Ensure the value appears in your .pypirc file. '
+         'Defaults to "pypi".'
+)
+args = parser.parse_args()
+
+# Move to the directory of this file.
+os.chdir(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
I don't understand what you mean, sorry. Using the cwd argument to subprocess.Popen doesn't mean you have to run this tool in a specific directory, it just means you don't change the current process's working directory for no good reason. Instead of doing 'os.chdir(pkgdir)' you pass 'cwd=pkgdir' to the two subprocess.call() calls.